Example/BonMot/BONSpecialGenerator.swift

```diff
@@ -0,0 +1,219 @@
+#!/usr/bin/env swift
+
+//
+//  BONSpecialGenerator.swift
+//  BonMot
+//
+//  Created by Zev Eisenberg on 7/16/15.
+//
+
+import Foundation
+import AppKit
+
+// Please keep this array sorted
+let specialCharacters: [unichar] = [
+    0x0009,
+    0x000A,
+    0x0020,
+    0x00A0,
+    0x2002,
+    0x2003,
+    0x2007,
+    0x2009,
+    0x200A,
+    0x200B,
+    0x2011,
+    0x2012,
+    0x2013,
+    0x2014,
+    0x2026, // HORIZONTAL ELLIPSIS
```

Comment:
![untitled](https://cloud.githubusercontent.com/assets/464574/11099685/c061bdea-887a-11e5-9e9e-4367731fa6ad.png)
That's just one more thing to get out of sync. The .m file is law - if it's right, pretty much nothing else matters. And that array is kept sorted in code just so it's easy to find where to insert new characters later.